src/item/Group.js

```diff
@@ -172,7 +172,7 @@ var Group = Item.extend(/** @lends Group# */{
         var clipItem = this._getClipItem();
         return clipItem
             ? clipItem._getCachedBounds(
-                matrix && matrix.appended(clipItem._matrix),
+                matrix && matrix.appended(clipItem._matrix) || clipItem._matrix,
```

Comment:
This is great! Just a detail (as so often with me :). This should be:

```js
matrix ? matrix.appended(clipItem._matrix) : clipItem._matrix
```

Reply:
And since we're now referencing twice, you could move it to a local variable for better minification:

```js
var clipItem = this._getClipItem(),
    _matrix = clipItem._matrix; 
...
matrix ? matrix.appended(_matrix) : _matrix
```

Reply:
Thanks, it's not a reflex yet ! :)